modules/ROOT/pages/install-openshift.adoc

```diff
@@ -0,0 +1,162 @@
+
+. In the Red Hat OpenShift console, navigate to **Operators** > **OperatorHub**.
+. In the OperatorHub search field, search for the Runtime Fabric operator.
+. In the rtf-agent-operator dialog, click **Install**.
+
+NOTE: Installing the Runtime Fabric operator requires manual approval and may take several minutes to complete.
+
+== Configure the Runtime Fabric operator
+
+To configure the Runtime Fabric operator, you supply the necessary values when prompted.
+ 
+. In the Red Hat OpenShift console, navigate to **Operators** > **Installed Operators**.
+. Select the Runtime Fabric operator.
+. In the console, click **Create Instance**, and select **Configure via form view**. 
+. Add any required parameters. Refer to the Installation Parameters Reference for guidance.
+. Click **Create**.
+
+== Installation Parameters Reference
+
+The following is an example YAML view of the installation parameters. 
+
+[source,yaml]
+--
+activationData: <activation data>
+OGQ=
+proxy:
+  http_proxy:
+  http_no_proxy:
+  monitoring_proxy:
+muleLicense: <mule license key>
+global:
+image:
+  rtfRegistry: <rtf-runtime-registry.kqa.msap.io or local registry value>
+    pullSecretName: rtf-pull-secret
+  containerLogPaths:
+  - /var/lib/docker/containers
+  - /var/log/containers
+  - /var/log/pods
+--
+
+=== Required Parameters
+
+The values for these required parameters are set when you create the Runtime Fabric in Runtime Manager. If you’re not using a local registry, use the default values for the registry URL and pull secret. 
+
+[%header%autowidth.spread]
+|===
+|Key |Value |Example
+|`activationData` |Activation data |YW55cG9pbnQubXVsZXNvZnQuY29tOjBmODdmYzYzLTM3MWUtNDU2Yy1iODg5LTU5NTkyNjYyZjUxZQ==
+|`rtfRegistry` |Registry URL  |`rtf-runtime-registry.kprod.msap.io`
+|`pullSecretName` |Registry secret |`<pull_secret>`
+|`muleLicense` |Mule license for applications |`<mule_license_key>`. Must be Base64 encoded.
+|===
+
+=== Optional Parameters
+Set these optional parameters as needed.
+
+[%header%autowidth.spread]
+|===
+|Key |Value |Example
+a|      
+      
+* `proxy.http_proxy`
+* `proxy.http_no_proxy` |Proxy and no_proxy values a| 
+
+* `http://<user>:<pass>@<10.0.0.1>:<8080>` 
+* `<1.1.1.1:8888,2.2.2.2:9999>`
+|`proxy.monitoring_proxy` | Anypoint Monitoring proxy values | `socks5://<user>:<pass>@<10.0.0.2>:<8080>`
+|`global.containerLogPaths` |The Filebeat read path a|
+
+* `/var/lib/docker/`
+* `/var/log/containers`
+* `/var/log/pods`
+|===
+
+include::partial$include-install-tasks.adoc[tags=licenseKey]
+include::partial$include-install-tasks.adoc[tags=ingressResource]
+include::partial$include-install-tasks.adoc[tags=validate]
+
+== Prepare for Deploying Mule Apps to Red Hat OpenShift
+
+Before you deploy any Mule applications to your Red Hat OpenShift cluster, perform the following steps. 
+
+Also, before attempting to deploy an Mule applications, ensure you've installed your Mule license key. 
```

Comment:
These two sentences could be merged to something like: 

_Before you deploy any Mule applications to your Red Hat OpenShift cluster, ensure you've installed your Mule license key and then, perform the following steps._ 

Or, the second sentence could be just converted to an additional step: 

_... perform the following steps._ 

_. Ensure you've installed your Mule license key._
_. Create a namespace..._

Reply:
I like that. Thanks